docs/current/cloud/572923-get-started.md

```diff
@@ -0,0 +1,313 @@
+---
+slug: /cloud/572923/get-started
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Get Started with Dagger Cloud
+
+## Introduction
+
+Dagger Cloud provides pipeline visualization, operational insights, and distributed caching for your Daggerized pipelines.
+
+This guide helps you get started with Dagger Cloud. Here are the steps you'll follow:
+
+- Sign up for Dagger Cloud
+- Connect Dagger Cloud with your CI provider or CI tool
+- Visualize CI runs with Dagger Cloud
+- Improve CI performance with Dagger Cloud's distributed caching
+
+The next sections will walk you through these steps in detail.
+
+## Prerequisites
+
+This guide assumes that:
+
+- You have an understanding of how Dagger works. If not, [read the Dagger Quickstart](../quickstart/index.mdx).
+- You have a GitHub account (required for Dagger Cloud identity verification). If not, [register for a free GitHub account](https://github.com/signup).
+- You have a source code repository and a Dagger pipeline that interacts with it. If not, follow the steps in Appendix A to [create and populate a GitHub repository with a sample application and Dagger pipeline](#appendix-a-create-a-dagger-pipeline).
+
+## Step 1: Sign up for Dagger Cloud
+
+:::info
+At the end of this step, you will have signed up for Dagger Cloud and obtained a Dagger Cloud token. If you already have a Dagger Cloud account and token, you may skip this step.
+:::
+
+Follow the steps below to sign up for Dagger Cloud, create an organization and obtain a Dagger Cloud token.
+
+1. Browse to the [Dagger Cloud website](https://www.dagger.io/cloud/signup). Click *Continue with GitHub* to log in with your GitHub account.
```

Comment:
Is this the correct start URL @d3rp3tt3 ?

Reply:
Ideally, we want to send people to the plans page so that they know what the various features and capabilities are of the plans before they sign up. We can add that when we're ready to release.

Reply:
Link fixed